src/components/ZetkinCalendar/index.tsx

```diff
@@ -51,10 +72,42 @@ const ZetkinCalendar = ({ baseHref, events, campaigns , tasks }: ZetkinCalendarP
         }
     };
 
+    const isTodayBeforeView = () =>{
+        if (focusDate.getFullYear() > dayjs().get('year')) return true;
```

Comment:
This is a small thing (that I learnt from @richardolsson XD ) but it's worth considering when or if you use single-line if statements. 
Although it's concise and arguably as easy to read, the indents in the code provide useful visual information about the complexity of branches

so seeing 
`if (focusDate.getFullYear() > dayjs().get('year')) {
        return true
  }`
  
visually indicates that there's an execution branch there, which is nice.